### PR TITLE
Mat 8569 button cql editor search result not working

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-8569](https://jira.cms.gov/browse/MAT-8569)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included and Code coverage is verified.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/cqllibraryservice/repositories/LibraryAclRepositoryImpl.java [hunk 1]
```diff
@@ -101,7 +101,8 @@ public List<LibraryListDTO> findLibrariesByNameAndModelOrderByNameAscAndVersionD
                     Criteria.where("draft").is(false),
                     Criteria.where("model").is(model)));
 
-    ProjectionOperation projectionOperation = project("cqlLibraryName", "version", "librarySet");
+    ProjectionOperation projectionOperation =
+        project("cqlLibraryName", "version", "librarySet", "librarySetId", "createdBy");
```
Comment (adongare) [inline src/main/java/gov/cms/madie/cqllibraryservice/repositories/LibraryAclRepositoryImpl.java]: are we using createdBy anywhere?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/cqllibraryservice/repositories/LibraryAclRepositoryImpl.java]: @adongare Yes in the frontend for owner.

Comment (adongare) [reply src/main/java/gov/cms/madie/cqllibraryservice/repositories/LibraryAclRepositoryImpl.java]: should we not use `librarySet.owner` instead?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/cqllibraryservice/repositories/LibraryAclRepositoryImpl.java]: @adongare Ok I changed to get it from librarySet.owner instead. Thanks Ashok!

Comment (adongare) [reply src/main/java/gov/cms/madie/cqllibraryservice/repositories/LibraryAclRepositoryImpl.java]: Sure, np!

**Review by adongare — APPROVED** [review]
